modules/01 JavaScript Programming/Assignments/8/answer.txt

```diff
@@ -0,0 +1,9 @@
+var bloodPressure = 70;
+var isNormal = false;
+isNormal = (bloodPressure > 80 && bloodPressure < 120) ? true : false; 
+
+console.log(isNormal);
+
+isNormal = (bloodPressure < 80 || bloodPressure > 120) ? true : false;
```

Comment:
you should've assigned the opposite result

Reply:
I sent a correction ..